src/include/storage/delta_record.h

```diff
@@ -137,13 +121,14 @@ class PACKED ProjectedRow {
   }
 
  private:
+  friend class ProjectedRowInitializer;
   uint32_t size_;
   uint16_t num_cols_;
   byte varlen_contents_[0];
 
-  uint32_t *AttrValueOffsets() { return reinterpret_cast<uint32_t *>(ColumnIds() + num_cols_); }
+  uint32_t *AttrValueOffsets() { return StorageUtil::AlignedPtr<uint32_t>(ColumnIds() + num_cols_); }
```

Comment:
This isn't a change in this PR, but reading this again, why isn't this + num_cols_ * sizeof(uint16_t)?